dev/update-examples-deps/main.go

```diff
@@ -24,16 +23,16 @@ func main() {
 				if dir == "." {
 					continue
 				}
-
-				fmt.Println("update of", file, "@", dir)
-
-				if err := updateDeps(dir, file); err != nil {
-					panic(err)
-				}
-
-				if err := updateWatermill(dir, file); err != nil {
-					panic(fmt.Sprintf("failed to update %s: %s", file, err))
-				}
+				//
+				//fmt.Println("update of", file, "@", dir)
+				//
+				//if err := updateDeps(dir, file); err != nil {
+				//	panic(err)
+				//}
+				//
+				//if err := updateWatermill(dir, file); err != nil {
+				//	panic(fmt.Sprintf("failed to update %s: %s", file, err))
+				//}
```

Comment:
The fact this is commented lead me to think that either:
- it's a debug that shouldn't be there
- it's a code that is no longer needed

I would leave it like this

Reply:
Sorry about this change. I forgot to revert in and that's how it sneaked in.
I rebased the PR so this change is removed.